src/nni_manager/common/utils.ts

```diff
@@ -146,6 +149,15 @@ function parseArg(names: string[]): string {
     return '';
 }
 
+function crossPlatformStringify(args:any):any{
```

Comment:
function name is confusing,  suggest rename to encodeCmdLineArgs or better name to describe it more precisely.